src/components/data/DataTable.js

```diff
@@ -386,6 +389,7 @@ const DataTable = props => {
                         extraActions: [
```

Comment:
Not specific to this PR, but we should probably hide these actions to edit the column when selecting data for a workflow.